src/btreemap/iter.rs

```diff
@@ -6,12 +6,14 @@ use crate::{types::NULL, Address, Memory, Storable};
 use std::borrow::Cow;
 use std::ops::{Bound, RangeBounds};
 
+#[derive(Clone)]
```

Comment:
Cloning is something we explicitly do not want to support. Cloning nodes/cursors is a heavy operation and we should avoid it.